release/stable/ubuntu14.04/meta.json

```
@@ -3,5 +3,9 @@
     "UseLinuxVersion": false,
     "PackageFormat": "powershell${previewTag}_${PS_VERSION}-1.ubuntu.14.04_amd64.deb",
     "osVersion": "Ubuntu 14.04",
+     "tagTemplates": [
+         "#psversion#-ubuntu-#tag#",
+         "ubuntu-#shorttag#"
+     ]
```

missing a command after the `]`
```suggestion
     ],
```